examples/some-verifier/frontend/src/components/Issuer.tsx

```diff
@@ -66,7 +71,7 @@ function Issuer() {
       const url = new URL(window.location.href);
       url.searchParams.set(Platform.Discord, 'true');
       window.history.replaceState(null, '', url);
```

Comment:
I think we should move this (and the corresponding piece of code for telegram issuing) to the wrapping `App` component and the respective callbacks passed into the `Issuer` component. I think it's more clear what happens, if 1 component interfaces with URL query params 🙂

Reply:
Good idea.